plugins/modules/license_keys.py

```diff
@@ -0,0 +1,291 @@
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
@rainerleber I'm happy to add a license header, but I saw that none of the other python files have a license header.

I think the headers of the files are very inconsistent anyways, some have shebangs, some not - also, I think the `coding: utf-8` is not necessary anymore for python 3.

I'd propose to:
- not touch the headers in this PR
- have a follow-up PR that aligns all files' headers:
  - remove any encoding header as it's not needed with python 3
  - remove any shebangs in the module_utils, add them to modules (only these are invoked directly)
  - I'd argue that the license header is not needed in any file since it's covered by the project's `LICENSE` in the root

Reply:
sounds good to me open a additional pr to align the license.